uvicorn/protocols/websockets/websockets_impl.py

```diff
@@ -231,6 +232,13 @@ async def asgi_send(self, message):
                 )
                 self.initial_response = None
                 self.accepted_subprotocol = message.get("subprotocol")
+                if "headers" in message:
+                    self.extra_headers.extend(
+                        # ASGI spec requires bytes
+                        # But for compability we need to convert it to strings
+                        (name.decode(), value.decode())
```

Comment:
It cannot be UTF-8. Most of the places we use latin-1, so I guess we'll go for it, but I'm confirming here: https://gitter.im/encode/community?at=61ba0a7cfd9244223e47b122

```suggestion
                        (name.decode("latin-1"), value.decode("latin-1"))
```